src/domains/Work/components/WorkEdit/presentations/items/AssetUpload.tsx

```diff
@@ -23,27 +24,10 @@ export const AssetUpload: FC<Props> = ({ handleUploadAssets, assets }) => (
       <Typography className="text-red-500" variant="body2">
         必須
       </Typography>
+      <InfoPopOver>
+        {`対応形式:\n    画像[.png .jpg .jpeg .bmp .gif]\n    動画[.mp4 .mov]\n    音源[.mp3 .wav .m4a ]\n    モデル[.gltf .fbx]\n    zip[.zip]`}
```

Comment:
文字の前後の空白を入れるか入れないかで統一してほしいです
`[.xxx]` or `[ .xxx ]`
前者なら以下の修正
```suggestion
        {`対応形式:\n    画像[.png .jpg .jpeg .bmp .gif]\n    動画[.mp4 .mov]\n    音源[.mp3 .wav .m4a]\n    モデル[.gltf .fbx]\n    zip[.zip]`}
```